documentation/custom-image-support/README.md

```diff
@@ -0,0 +1,78 @@
+
+### FROM Statement
+
+If you are wanting to build a Docker image to utilize ADE deployments and access the ADE CLI, you will want to base your image off of one of the ADE-authored images. This can be done by including a FROM statement within a created DockerFile for your new image pointing to an ADE-authored image hosted on Microsoft Artifact Registry.
+
+Here's an example of that FROM statement, pointing to the ADE-authored core image:
+```docker
+FROM mcr.microsoft.com/deployment-environments/runners/core:latest
+```
+
+This statement pulls the most recently-published core image, and makes it a basis for your custom image.
+
+### RUN Statement
+
+Next, you can use the RUN statement to install any additional packages you would need to use within your image. ADE-authored images are based off of the Azure CLI image, and have the ADE CLI and JQ packages pre-installed. You can learn more about the Azure CLI [here](https://learn.microsoft.com/en-us/cli/azure/), and the JQ package [here](https://devdocs.io/jq/).
+
+Here's an example used in our Bicep image, installing the Bicep package within our Dockerfile.
+```docker
+RUN az bicep install
+```
+
+### Executing Operation Shell Scripts
+
+Within the ADE-authored images, operations are determined and executed based off of the operation name. Currently, the two operation names supported are 'deploy' and 'delete', with plans to expand this moving forward.
+
+To set up your custom image to utilize this structure, specify a folder at the level of your Dockerfile named 'actions.d', and specify two files, 'deploy.sh', and 'delete.sh'. The 'deploy' shell script will run when your environment is created or redeployed, and the 'delete' shell script will run when your environment is deleted. You can see examples of this within this repository under the Runner-Images folder for the ARM-Bicep image.
+
+To ensure these shell scripts are executable, add the following lines to your Dockerfile:
+```docker
+COPY actions.d/* /actions.d/
+RUN find /actions.d/ -type f -iname "*.sh" -exec dos2unix '{}' '+'
+RUN find /actions.d/ -type f -iname "*.sh" -exec chmod +x {} \;
+```
+### Building the Image
+
+To build the image to be pushed to your registry, please ensure the Docker Engine is installed on your computer, navigate to the directory of your Dockerfile, and run the following command:
+```
+docker build . -t {YOUR_REGISTRY}.azurecr.io/{YOUR_IMAGE_LOCATION}:{YOUR_TAG}
+```
+
+For example, if you wanted to save your image under a repository within your repo named 'customImage', and you wanted to upload with the tag version of '1.0.0', you would run:
+
+```
+docker build . -t {YOUR_REGISTRY}.azurecr.io/customImage:1.0.0
+```
+## Pushing the Docker Image to a Registry
+In order to currently use custom images, you will need to set up a publicly-accessible image registry with anonymous image pull enabled. This way, Azure Deployment Environments can access your custom image to execute in our container.
```

Comment:
Slightly weird verbiage with "currently"

Reply:
Removed the use of 'currently'